pillar/base/firewall/fastly-logging.sls

```diff
@@ -4,12 +4,14 @@
 firewall:
 {% for address in fastly_ip_ranges.get('dict', {}).get('addresses', []) %}
   fastly_syslog_ipv4_{{ loop.index }}:
-    source: {{ address }}
+    source:
+      - {{ address }}
```

Comment:
The new rule for salt-lint is suggesting that nested dictionaries should be over indented. So in this case we should add two-spaces before `source`